styles2.css

```diff
@@ -22,6 +22,11 @@ body{
     
 }
 
+.logo h1 {
+    font-size: 28px;
+    padding: 8%;
+}
+
```

Comment:
## Style your logo
Please help styling logo with red color and try checking for the right font, as this will help improve the quality of looks of the website.

## Add Href
Please add an href to your logo that links to home, that will make it easier for users to navigate and is commonly used and good web practice.

Reply:
[OPTIONAL] Kudos for the job well done👍. I think it will be good if you can include emojis ☺ in your reviews! 🤖 They make the code review more human-friendly.